### DISCUSSION TITLE
Subteams and Team Members

### DISCUSSION THREAD

**Discussion by NatiPerez459** [uprm-inso4101-2024-2025-s1/semester-project-mercado-colegial-online#13] [General]
An updated list of teams and team members has been created in another discussion. The discussion is named Updated Subteams and Team Members 

The following were the initial teams, although never implemented.
--------------------------------------------------------------------------------------------------------------------------------------------------------

Here is a list of subteams created for this project so that you may begin developing your Issues following the SMART and FAST criteria. Role is assigned next to the name.

Subteams:

Seller’s Inventory Tracker - 4
Alvin Badillo (Team Lead)
Jadhiel Montero (Backend)
Edyan Cruz (Frontend)
Francisco Rovira (Infrastructure Developer)

Merchant/Buyer Login - 3
Juan Iranzo (Backend)
Fernando Hidalgo (Frontend)
Christian Rodriguez (Team Lead)

Shopping Cart and Wishlist Functionality - 4
Jonael Garcia (Frontend)
Joshua Davila (Backend)
Christian Barreras (Team Lead)
Luis Hernandez (Infrastructure Developer)

Merchant's Products Feed - 4
Fabiola Torres (Team Lead)
Yadziel Perez (Frontend)
Marcelo Perez (Backend)
Elliot Gomez( Infrastructure Developer)

Checkout Process - 4
Ryan Velez (Team Lead)
Julio Camacho (Frontend)
Diego Vazquez (Backend)
Gabriel Calvente (Infrastructure Developer)

Seller’s Inventory Database - 2
Yampier Fernandez (Team Lead - Database Developer)
Sebastian Montalvo (Database Developer)

Individual Product Feed - 3
Karina Lopez (Frontend)
Michael Martinez (Backend)
Jose Covas (Team Lead - Infrastructure Developer)

**Comment by Rezanmonis** [comment]
We are looking for frontend and backend developers as our teammates for our edtech project rezanmonis@gmail.com
